packages/backend/server/src/modules/payment/resolver.ts

```diff
@@ -189,12 +189,14 @@ export class SubscriptionResolver {
   async checkout(
     @CurrentUser() user: User,
     @Args({ name: 'recurring', type: () => SubscriptionRecurring })
-    recurring: SubscriptionRecurring
+    recurring: SubscriptionRecurring,
+    @Args('idempotencyKey') idempotencyKey: string
```

Comment:
the checkout session is ok to not be non-idempotent

Reply:
stripe.customers.create need to be idempotent